dkim/verify.go

```diff
@@ -86,11 +86,19 @@ type signature struct {
 	v string
 }
 
+type VerifierOptions struct {
+	LookupTXT txtLookupFunc
+}
+
 // Verify checks if a message's signatures are valid. It returns one
 // verification per signature.
 //
 // There is no guarantee that the reader will be completely consumed.
 func Verify(r io.Reader) ([]*Verification, error) {
+	return VerifyWithOption(r, nil)
+}
+
+func VerifyWithOption(r io.Reader, option *VerifierOptions) ([]*Verification, error) {
```

Comment:
For consistency's sake, can we rename the function to `VerifyWithOptions` and the argument to `options` (plural)?

Reply:
fixed